spec/features/checkout_spec.rb

```diff
@@ -163,6 +163,7 @@ def fill_in_address
   end
 
   def use_address_full_name?
+    SolidusSupport.combined_first_and_last_name_in_address? ||
```

Comment:
I think that if we use `SolidusSupport.combined_first_and_last_name_in_address?` we can skip the other conditions `Spree::Config.has_preference?(:use_combined_first_and_last_name_in_address) && Spree::Config.use_combined_first_and_last_name_in_address` because they are already checked in the method itself.

Reply:
I tried to recreate an issue I had earlier with this statment, but could not reproduce the error so I have updated this block to only include `SolidusSupport.combined_first_and_last_name_in_address?`